.github/workflows/schedule.yml

```diff
@@ -0,0 +1,151 @@
+name: SCHEDULED
+on:
+  workflow_dispatch:
+  schedule:
+    #  Exectues e2e compatibility test every Wednesday at 1:00 AM and if you need to convert to UTC+8 timezone, you need to -8 hours for cron expression
+    - cron: "0 17 ? * 2"
```

Comment:
What's the meaning of `?` here? Seems only `*` is allowed?  @wlq1212 
ref: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule